src/components/pages/startups/hero/contact-form/contact-form.jsx

```diff
@@ -99,38 +104,57 @@ const ContactForm = () => {
     e.preventDefault();
     const { firstname, lastname, email, companyWebsite, investor } = data;
     const loadingAnimationStartedTime = Date.now();
+    const values = [
+      {
+        name: 'firstname',
+        value: firstname,
+      },
+      {
+        name: 'lastname',
+        value: lastname,
+      },
+      {
+        name: 'email',
+        value: email,
+      },
+      {
+        name: 'company_website',
+        value: companyWebsite,
+      },
+      {
+        name: 'accelerator_private_investor',
+        value: investor,
+      },
+    ];
     setIsBroken(false);
     setFormState(FORM_STATES.LOADING);
 
     try {
       const response = await sendHubspotFormData({
         formId: HUBSPOT_STARTUPS_FORM_ID,
         context,
-        values: [
-          {
-            name: 'firstname',
-            value: firstname,
-          },
-          {
-            name: 'lastname',
-            value: lastname,
-          },
-          {
-            name: 'email',
-            value: email,
-          },
-          {
-            name: 'company_website',
-            value: companyWebsite,
-          },
-          {
-            name: 'accelerator_private_investor',
-            value: investor,
-          },
-        ],
+        values,
       });
 
       if (response.ok) {
+        const eventName = 'Startup Form Submitted';
+        const eventProps = {
+          email,
+          first_name: firstname,
+          last_name: lastname,
+          company_website: companyWebsite,
+          investor,
+        };
+        if (isFormDataSentToCustomerIo) {
+          try {
```

Comment:
We want to fail silently here without blocking the user since the data is still submitted to HubSpot